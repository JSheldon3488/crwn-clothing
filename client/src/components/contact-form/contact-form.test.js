import React from 'react';
import { shallow } from 'enzyme'
import ContactForm from './contact-form.component'


describe('ContactForm Component Tests', () => {
    const wrapper = shallow(<ContactForm />)

    it('should render and match snapshot', () => {
        expect(wrapper).toMatchSnapshot();
    })
})