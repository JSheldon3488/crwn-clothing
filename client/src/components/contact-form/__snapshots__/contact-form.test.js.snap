// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContactForm Component Tests should render and match snapshot 1`] = `ShallowWrapper {}`;
